Notation.Plugin.AzureKeyVault/Command/GetPluginMetadata.cs

```diff
@@ -5,9 +5,12 @@ namespace Notation.Plugin.AzureKeyVault.Command
     /// <summary>
     /// Implementation of get-plugin-metadata command.
     /// </summary>
-    public class GetPluginMetadata : IPluginCommand
+    public partial class GetPluginMetadata : IPluginCommand
     {
-        public const string Version = "1.0.0";
+        public static readonly string Version = "Unknown";
+        public static readonly string CommitTime = "Unknown";
+        public static readonly string CommitHash = "Unknown";
```

Comment:
nit:
```suggestion
        public static readonly string Version = "1.0.0+unreleased";
        public static readonly string CommitTime = "";
        public static readonly string CommitHash = "";
```

Reply:
Updated. The default value is set in `.csproj`